### PR TITLE
fix smashystream

### PR DESCRIPTION
**Pull request by ztpn**
Fixed smashystream
Fixed smashystream-f/video1 embed
Removed smashystream-d/dued because it has dubed content, in hindi language
Add smashystream-o/opstream

 - [x] I have read and agreed to the [code of conduct](https://github.com/movie-web/movie-web/blob/dev/.github/CODE_OF_CONDUCT.md).
 - [x] I have read and complied with the [contributing guidelines](https://github.com/movie-web/movie-web/blob/dev/.github/CONTRIBUTING.md).
 - [ ] What I'm implementing was assigned to me and is an [approved issue](https://github.com/movie-web/movie-web/issues?q=is%3Aopen+is%3Aissue+label%3Aapproved). For reference, please take a look at our [GitHub projects](https://github.com/movie-web/movie-web/projects).
 - [x] I have tested all of my changes.

### CODE REVIEW

File: src/providers/embeds/smashystream/video1.ts [hunk 1]
```diff
@@ -1,32 +1,50 @@
...
 
 type FPlayerResponse = {
   sourceUrls: string[];
-  subtitleUrls: string;
+  subtitles: string | null;
 };
 
+// if you don't understand how this is reversed
+// check https://discord.com/channels/871713465100816424/1186646348137775164/1225644477188935770
+// feel free to reach out atpn or ciaran_ds on discord if you've any problems
```
Comment (Ciarands) [inline src/providers/embeds/smashystream/video1.ts:13]: woah he has my name

File: src/providers/all.ts [hunk 2]
```diff
@@ -85,8 +82,5 @@ export function gatherAllEmbeds(): Array<Embed> {
     streamvidScraper,
     voeScraper,
     streamtapeScraper,
-    droploadScraper,
```
Comment (JorrinKievit) [inline src/providers/all.ts:88]: Why are these removed?

**Review by JorrinKievit — CHANGES_REQUESTED** [review]
Code looks good, but embed.smashystream.com seems down. URL's seem to 404: https://embed.smashystream.com/video1dn.php?tmdb=299534

File: src/providers/all.ts (hunk 2 above)
Comment (ztpn) [reply src/providers/all.ts]: I was making changes on the wrong branch, I've fixed it now

**Comment by ztpn** [comment]
> Code looks good, but embed.smashystream.com seems down. URL's seem to 404: https://embed.smashystream.com/video1dn.php?tmdb=299534

Works for me, try 823464

**Review by JorrinKievit — APPROVED** [review]
